third_party/clang_llvm_x86_64_linux_gnu_ubuntu.BUILD

```diff
@@ -67,23 +84,37 @@ cc_toolchain_import(
     name = "llvm_libcxx",
     hdrs = glob(["include/c++/v1/**"]),
     includes = ["include/c++/v1"],
-    static_library = "lib/libc++.a",
+    static_library = "lib/x86_64-unknown-linux-gnu/libc++.a",
     target_compatible_with = select({
         "@platforms//os:linux": ["@platforms//cpu:x86_64"],
         "//conditions:default": ["@platforms//:incompatible"],
     }),
     visibility = ["@rules_cc_toolchain//config:__pkg__"],
     deps = [
+        # TODO: Add more indirection.
+        ":llvm_config_site",
+        # ":llvm_libstddef",
         "@rules_cc_toolchain_config//:libc",
         "@rules_cc_toolchain_config//:libunwind",
     ],
 )
 
+cc_toolchain_import(
+    name = "llvm_config_site",
+    hdrs = ["include/x86_64-unknown-linux-gnu/c++/v1/__config_site"],
+    includes = ["include/x86_64-unknown-linux-gnu/c++/v1"],
+    target_compatible_with = select({
+        "@platforms//os:linux": ["@platforms//cpu:x86_64"],
+        "//conditions:default": ["@platforms//:incompatible"],
+    }),
+    visibility = ["@rules_cc_toolchain//config:__pkg__"],
+)
+
 cc_toolchain_import(
     name = "llvm_libcxx_abi",
     hdrs = glob(["include/c++/v1/**"]),
     includes = ["include/c++/v1"],
-    static_library = "lib/libc++abi.a",
+    static_library = "lib/x86_64-unknown-linux-gnu/libc++abi.a",
```

Comment:
Is this the new location for libc++? I'm surprised it's not a stable path...

Reply:
Sadly, so it seems :/. `find -L . -name libc++abi.a` finds only this one file.